package.json

```diff
@@ -62,6 +62,7 @@
         "onCommand:extension.helmDepUp",
         "onCommand:extension.helmInspectValues",
         "onCommand:extension.helmGet",
+	    "onCommand:extension.helmPackage",
```

Comment:
Can you check the indentation on this please?  It looks a bit off but I'm not sure if that's GitHub preview.

Reply:
it seems to be an issue with Github preview as the file is intended as excepted as you can see in [here](https://github.com/mostafahussein/vscode-kubernetes-tools/blob/feature-1/package.json#L65) however it looks different in this preview. Also I tried to use the github editor itself online but still looks different in the preview

Reply:
Also through VSCode itself looks indented correclty

Reply:
No worries, I wonder if it's some tabs vs spaces nonsense, but let's not get hung up on it; thanks for taking a look.  I'll get the PR merged and if it doesn't look right to some people then we can sort it in one of our formatting passes.  Many thanks for the contribution!

Reply:
For what it's worth, it was indeed mixed tabs/spaces (which is why it showed up off in the GH preview but looked fine in VS Code).  Normally the linter flags this stuff but I guess it doesn't know how to check `package.json`.  Anyway all sorted now - thanks for your patience!